docs/conf.py

```diff
@@ -365,7 +365,6 @@ def linkcode_resolve(domain, info):
 
 # Generate redirects from deleted files to new sources
 rediraffe_redirects = {
-    'notebooks/quickstart.md': 'quickstart.md',
```

Comment:
We should keep this, and also add an entry for `quickstart.md` that maps to the new doc.

The purpose of this is so that we don't break hyperlinks when we remove pages from the website.